test/UnitTest/TestingControllers/DataControllerUnitTests.cs

```diff
@@ -172,11 +172,11 @@ private static bool VerifyPropertyListInput(int expectedPropCount, List<string>
             });
 
             blobRepositoryMock
-                .Setup(d => d.ReadBlob(It.IsAny<string>(), It.IsAny<string>()))
+                .Setup(d => d.ReadBlob(It.IsAny<string>(), It.IsAny<string>(), null))
```

Comment:
Please use proper mock syntax.
```suggestion
                .Setup(d => d.ReadBlob(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<int?>))
```
Fix all occurances.

Reply:
Done